drivers/SmartThings/matter-sensor/fingerprints.yml

```diff
@@ -54,6 +54,12 @@ matterManufacturer:
     vendorId: 0x141E
     productId: 0x0002
     deviceProfileName: contact-battery
+    # LeTianpai
+  - id: "LeTianPai/Motion"
+    deviceLabel: "LeTianPai Presence Sensor Box"
```

Comment:
You can omit the quotes around the deviceLabel.

Reply:
Thanks, the quote has been omitted